### DISCUSSION TITLE
Getting seq unavailable after changing a config and restarting service

### DISCUSSION THREAD

**Discussion by psundareshan** [datalust/seq-tickets#1354] [Help]
Hi,

I had to update canonicalUri and restarted the seq service which is hosted and running on a windows server. I'm getting below error now, please help.

{"Error":"Seq is unavailable. Failed to initialize storage: Flare native storage failed (InternalError), data file `\"C:\\\\ProgramData\\\\Seq\\\\Stream\\\\stream.08d94e48d6eb6a00_08d94e4ba22ce200.38a346d7436a49c38d5138689eb48a6b.span\"` is missing."}

Kind regards,
Prashanth
psundareshan@guildlink.com.au

**Comment by nblumhardt** [comment]
Hi Prashanth,

Thanks for getting in touch. The repair tool should be bundled with the version you're running (if you can let me know the exact version, I can confirm).

From an administrative PowerShell prompt:

```
seq service stop
& "C:\Program Files\Seq\Native\flaretl.exe" repair --metadata "C:\ProgramData\Seq\Stream"
seq service start
```

If this fails, please let me know :-)

It would also be great to have any info on the kind of disk Seq is using for storage; normally these kinds of errors appear on network filesystems (e.g. Azure Files), but it's also possible something else was involved. Seq 2021.2 has been hardened a bit more against these kinds of issues, if you're able to update at some point.

**Reply by CoderBrad** [reply]
This failed for me. The repair command failed with a message about needing a .flare file. The command I ended up using was:
```
seq service stop
& "C:\Program Files\Seq\Native\flaretl.exe" repair --metadata "C:\ProgramData\Seq\Stream\stream.flare"
seq service start

**Reply by IanElsinga** [reply]
This was helpful, thanks. I was able to use this technique to copy stream events from one seq box to a new, empty one without stopping the source seq service.  After copying I just ran this command and the new one fired up without issue and with all the events.
